parm/snow/obs/config/bufr_sfcsno_mapping.yaml

```diff
@@ -65,11 +62,18 @@ encoder:
       coordinates: "longitude latitude"
       source: variables/stationIdentification
       longName: "Identification of Observing Location"
-      units: "m"
+      units: "index"
```

Comment:
@jiaruidong2017 The unit of stationIdentification is not defined (empty) in the IODA convention table, and it is left for users to decide.  Have you discussed the unit for `stationIdentification` with  JCSDA (Ben Ruston)?
Let's leave it as "index" for now and I will open an issue in IODA repository and check if we can use "index" as unit for this variable.